linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_data_retention_space_left_percentage/ansible/shared.yml

```diff
@@ -0,0 +1,15 @@
+# platform = Red Hat Virtualization 4,multi_platform_fedora,multi_platform_rhel,multi_platform_ol,multi_platform_sle
```

Comment:
The rule has `prodtype:  fedora,rhel7,rhel8,rhel9` so I think that the remediation shouldn't have RHV, SLE and Oracle Linux.  I would prefer `platform = multi_platform_all`, the remediation script won't get into other products because the rule has `prodtype`.